### PR TITLE
Yuri/squarelotron

### PR DESCRIPTION
**Pull request by yvayne**
image

### CODE REVIEW

File: src/main/java/org/minions/devfund/yuri/Squarelotron.java [hunk 1]
```diff
@@ -0,0 +1,197 @@
...
+  public int[][] getMatrix() {
+    return matrix.clone();
+  }
+
+  /**
+   * Gets the matrix size.
+   *
+   * @return size.
+   */
+  public int getSize() {
```
Comment (carledriss) [inline src/main/java/org/minions/devfund/yuri/Squarelotron.java]: This method is not necessary.
Remove it!

File: src/main/java/org/minions/devfund/yuri/Squarelotron.java [hunk 2]
```diff
@@ -0,0 +1,197 @@
...
+      }
+    }
+  }
+
+  /**
+   * Gets the rings amount of a matrix.
+   *
+   * @return the rings amount;
+   */
+  public int getRingsAmount() {
```
Comment (carledriss) [inline src/main/java/org/minions/devfund/yuri/Squarelotron.java]: Is it really needed this method ?
Consider removing it

File: src/main/java/org/minions/devfund/yuri/Squarelotron.java (hunk 1 above)
Comment (yvayne) [reply src/main/java/org/minions/devfund/yuri/Squarelotron.java]: done

File: src/main/java/org/minions/devfund/yuri/Squarelotron.java (hunk 2 above)
Comment (yvayne) [reply src/main/java/org/minions/devfund/yuri/Squarelotron.java]: done

**Comment by carledriss** [comment]
LGTM! 👍

**Pull request merged**